### PR TITLE
Add a helper to strip HTML tags

### PR DESCRIPTION
**Pull request by LaurentGoderre**

### CODE REVIEW

**Comment by jonschlinkert** [comment]
@LaurentGoderre can you do this PR against the v0.6.0 branch? we're going to be merging any day now. thx!

**Comment by LaurentGoderre** [comment]
@jonschlinkert will we still be able to use it on v0.4.x?

**Comment by jonschlinkert** [comment]
hmm, if we merge into the current branch, would you be okay with doing another pr afterwards to merge into v0.6.0? we wouldn't be able to merge it otherwise, the versions are too different.  @doowb any problems with that?

**Comment by LaurentGoderre** [comment]
Absolutely

**Comment by doowb** [comment]
That sounds good to me.

File: lib/helpers/helpers-html.js [hunk 1]
```diff
@@ -157,6 +157,16 @@ module.exports.register = function (Handlebars, opts) {
...
       return new Handlebars.SafeString(figure);
+    },
+
+    /**
+     * {{#striptags}}{{{data}}}{{{//striptags}}}
+     * Remove all HTML tags from the nested context
+     * @param  {[type]} options [description]
+     * @return {[type]}         Text with HTML tags removed
+     */
+    striptags: function (options) {
```
Comment (LaurentGoderre) [inline lib/helpers/helpers-html.js]: @doowb @jonschlinkert is options right her or should it be context? I never know

Comment (doowb) [inline lib/helpers/helpers-html.js]: `options` are what handlebars passes as the last argument automatically. If you're planning on passing in a `context` when using the helper in a template, then you want to do `striptags: function (context, options) {`.

Your test below says `{{#striptags context options}}` but it's not actually testing that.

Comment (LaurentGoderre) [inline lib/helpers/helpers-html.js]: Fixed

Comment (jonschlinkert) [inline lib/helpers/helpers-html.js]: :+1:  to add a little more to what @doowb mentioned (b/c it took me forever to remember these), when you do `console.log()` on those objects, if you see `hash`, it's really the `options` object that is created by Handlebars, with additional properties that handlebars creates, like `root`. But context is the actual data object that is being built-up from various sources to pass to your templates. Both of these objects can be merged inside the helper function. Hope that helps (or maybe it made it worse lol)

Comment (jonschlinkert) [inline lib/helpers/helpers-html.js]: don't merge this just yet, give me a minute I'll add some comments. I think we need to make this a little more robust.

Comment (jonschlinkert) [inline lib/helpers/helpers-html.js]: maybe we could just get a couple more unit tests before merging to ensure it won't hang on unclosed tags or code comments. does that work for you guys?

Comment (LaurentGoderre) [inline lib/helpers/helpers-html.js]: When you unclosed tags do you mean `<a this is a link` or `<a href="">this is a link`?

Comment (jonschlinkert) [inline lib/helpers/helpers-html.js]: exactly, the first example.

**Comment by doowb** [comment]
There's a [sanitize](https://github.com/helpers/handlebars-helpers#sanitize) helper that does this for a passed in string.

If you still need something like this for block helpers, please submit a new PR since this was against an older version.

**Pull request closed without merging**